### PR TITLE
Fixes #1423. Indents fixed

### LINKED ISSUE #1423: Improve code formatting
- Some tests starts with excessive space ` // Copyright (c)...`
- Fix indents in a Set literals tests

### PR DESCRIPTION
**Pull request by sgrekhov**
Only indents fixed. This PR mainly result of "find and replace" command

### CODE REVIEW

**Review by eernstg — APPROVED** [review]

File: LanguageFeatures/Set-literals/set_literals_A04_t02.dart [hunk 1]
```diff
@@ -13,85 +13,85 @@ class C {}
...
+//                                ^
+// [cfe] A value of type 'Set<C>' can't be returned from a function with return type 'LinkedHashSet<C>'.
   LinkedHashSet<C> test12() => const <C>{};
-  //                           ^^^^^^^^^^^
-  // [analyzer] COMPILE_TIME_ERROR.RETURN_OF_INVALID_TYPE
-  // [cfe] A value of type 'Set<C>' can't be returned from a function with return type 'LinkedHashSet<C>'.
+//                             ^^^^^^^^^^^
+// [analyzer] COMPILE_TIME_ERROR.RETURN_OF_INVALID_TYPE
+// [cfe] A value of type 'Set<C>' can't be returned from a function with return type 'LinkedHashSet<C>'.
   LinkedHashSet<C> test13() => <C>{new C()}
```
Comment (eernstg) [inline LanguageFeatures/Set-literals/set_literals_A04_t02.dart:70]:
```suggestion
  LinkedHashSet<C> test13() => <C>{new C()};
```

Comment (eernstg) [reply LanguageFeatures/Set-literals/set_literals_A04_t02.dart:70]: Might as well fix this semicolon, too.

**Review by eernstg — COMMENTED** [review]
LGTM except the semicolon.

File: LanguageFeatures/Set-literals/set_literals_A04_t02.dart (hunk 1 above)
Comment (sgrekhov) [reply LanguageFeatures/Set-literals/set_literals_A04_t02.dart]: Thank you. Added this change

**Review by eernstg — APPROVED** [review]
LGTM.

As an aside, I created https://github.com/dart-lang/co19/issues/1425 in order to remind us that there are some syntax errors which do no seem to be the purpose of some tests.

**Pull request merged**